configs/katello_36.yaml

```diff
@@ -0,0 +1,181 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+  :3.5.0:
+    :redmine_release_id: 250
+  :3.5.1:
+    :redmine_release_id: 329
+:releases:
+  :3.6.0:
+    :redmine_release_id: 284
+:code_name: Imperial IPA
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-3.6
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :katello-selinux:
+    :branch: 3.0.3
+    :repo: https://github.com/Katello/katello-selinux.git
+  :katello-installer:
+    :branch: KATELLO-3.6
+    :repo: https://github.com/Katello/katello-installer.git
+    :version_branch: true
+  :hammer-cli-katello:
+    :branch: 0.11.4
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :foreman-packaging:
+    :branch: rpm/1.17
+    :repo: https://github.com/theforeman/foreman-packaging.git
+    :version_branch: true
+  :katello-agent:
+    :branch: 3.1.0
+    :repo: https://github.com/Katello/katello-agent.git
+  :bastion:
+    :branch: 6.1.0
+    :repo: https://github.com/Katello/bastion
+  :puppet-pulp:
+    :branch: 5.1.0
+    :repo: https://github.com/Katello/puppet-pulp
+  :puppet-foreman_proxy_content:
+    :branch: 6.0.0
+    :repo: https://github.com/Katello/puppet-foreman_proxy_content
+  :puppet-katello:
+    :branch: 7.0.0
+    :repo: https://github.com/Katello/puppet-katello
+  :puppet-candlepin:
+    :branch: 4.0.0
+    :repo: https://github.com/Katello/puppet-candlepin
+  :puppet-certs:
+    :branch: 3.0.1
+    :repo: https://github.com/Katello/puppet-certs
```

Comment:
There version numbers here for `puppet-*` may likely change. I don't think @ekohl has looked at releases for them just yet.

Reply:
Should I get rid of this for now then?

Reply:
Doing releases is still on my list. I know there will be some major version bumps so for now I'd point to master versions

Reply:
This will be true of most of these versions, afaik. i.e. We'll probably do a 0.12.z release of hammer-cli-katello to go along with 3.6/1.17.